jc/parsers/amixer.py

```diff
@@ -0,0 +1,205 @@
+from typing import List, Dict
```

Comment:
Missing the parser module docstring at the top. This is where the generated documentation, including the schema, comes from.

Reply:
Added, Resolved.